eslint.config.mjs

```diff
@@ -0,0 +1,81 @@
+import globals from "globals";
+import importPlugin from "eslint-plugin-import";
+import reactPlugin from "eslint-plugin-react";
+//import hooksPlugin from 'eslint-plugin-react-hooks';
+import jsxA11yPlugin from "eslint-plugin-jsx-a11y";
+import typescriptParser from "@typescript-eslint/parser";
+import typescriptPlugin from "@typescript-eslint/eslint-plugin";
+
+// Add eslint-plugin-react-hooks back when this issue is solved: https://github.com/facebook/react/issues/28313
```

Comment:
Hier nog een ticket voor maken dat we deze later nog een keer checken?

Reply:
Heb er een aangemaakt: https://github.com/nl-portal/nl-portal-frontend-libraries/issues/193